shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -121,6 +122,11 @@ void UpdateAssetResolverByType(std::unique_ptr<flutter::AssetResolver> updated_a
   flutter::Settings settings_;
 };
 
+BOOL BlurRadiusEqualToBlurRadius(CGFloat radius1, CGFloat radius2) {
+  const CGFloat epsilon = 0.01;
+  return radius1 - radius2 < epsilon;
```

Comment:
This doesn't seem quite right. Going past negative epsilon won't be caught. `(radius1 > (radius2 - epsilon)) && (radius1 < (radius2 + epsilon))` perhaps. Or if you are passing in a potentially zero epsion, check for direct equality too.

Reply:
Done

Reply:
GitHub didn't update. Did you miss staging this?

Reply:
fixed